tests/e2e/testing-manifests/tkg/tkg.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: run.tanzu.vmware.com/v1alpha1
```

Comment:
pick it from the same place which gc automation uses instead of hardcoding, this will help in reducing maintainence

Reply:
I'm using this API version for our testing at present.

Reply:
I am talking about the whole yaml

Reply:
In a VMC environment, we/customers are supposed to create a TKC and this is the YAML I have used to for VMC IA release testing.